agenta-web/src/lib/Types.ts

```diff
@@ -17,6 +17,14 @@ export interface Variant {
     parameters: Record<string, string> | null // parameters of the variant. Only set in the case of forked variants
 }
 
+// Define the interface for the tabs item in playground page
+export interface PlaygroundTabsItem {
+    key: string;
+    label: string;
+    children: JSX.Element;
+    closable: boolean;
```

Comment:
**Suggestion:** Consider adding comments to the `PlaygroundTabsItem` interface to explain what each property is used for. This will make the code easier to understand for other developers.
```suggestion
export interface PlaygroundTabsItem {
    // The unique key for the tab
    key: string;
    // The label to be displayed on the tab
    label: string;
    // The content to be displayed when the tab is selected
    children: JSX.Element;
    // Whether the tab can be closed
    closable: boolean;
}
```